src/test/resources/pages/instructorSearchPageSearchFeedbackResponseComments.html

```diff
@@ -216,4 +216,4 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
@wkurniawan07 is right! maybe it's because of #7312. I am responsible for that. But now it's fixed.
sorry about that!